src/templates/admin/workflow/manage_article_workflow.html

```diff
@@ -45,6 +48,20 @@
                     </table>
                 </form>
             </div>
+            <div class="large-6 columns">
+                <div class="title-area">
+                    <h2>Archive</h2>
+                </div>
+                {% if article.stage == 'Archived' %}
+                <p>This article has already been archived.</p>
+                {% else %}
+                <p>Archiving an article will remove it from the workflow, essentially tombstoning it. You can still access it through the Article Archive.</p>
+                <form method="POST">
+                    {% csrf_token %}
+                    <button name="archive" class="danger button">Archive Article</button>
```

Comment:
It would be good to use Joe's new `ConfirmableForm` here

Reply:
Will do so in 1.5.1: https://github.com/BirkbeckCTP/janeway/issues/3150